docs/minikube.md

```diff
@@ -0,0 +1,49 @@
+# Minikube
+
+This document details Minikube specific deployment instructions.
+Minikube is not recommended as a platform to test or develop Contour due to its network limitations.
+
+## Access your cluster
+
+On Minikube you can retrieve the external address of Contour's load balancer:
+```
+$ minikube service -n heptio-contour contour --url
+http://192.168.99.100:30588
+```
+
+## Configuring DNS
+
+On Minikube, you can fake DNS by editing `/etc/hosts` or you can use the provided example and not have to modify dns on your local machine.
```

Comment:
Reword slightly (context makes it clear, but you'll need to adjust the wording somewhat bc I'm not completely clear about the Minikube issues):
"Because of known issues with how Minikube manages DNS, we recommend that you run the provided example so that you do not have to modify DNS on your local machine. Otherwise, you must fake DNS by editing `/etc/hosts`."

(maybe it's Minikube and ports? Or something else? It should be short here; the explanation in ##Troubleshooting is clear but long and I'm trying to avoid linking bc usability.)